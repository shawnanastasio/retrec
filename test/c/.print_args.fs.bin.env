{
    "_" : "print_args.fs.bin",
    "SHELL" : "/bin/bash",
    "RETREC_ENVIRONMENT_WORKS" : "YES"
}
