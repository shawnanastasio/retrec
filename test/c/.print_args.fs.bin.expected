argc=1
-------
argv:
c/print_args.fs.bin
-------
-------
envp:
_=print_args.fs.bin
SHELL=/bin/bash
RETREC_ENVIRONMENT_WORKS=YES
-------
